iso15118/shared/messages/datatypes.py

```diff
@@ -45,12 +48,15 @@ def validate_value_range(cls, values):
         value = values.get("value")
         multiplier = values.get("multiplier")
         calculated_value = value * 10**multiplier
-        if calculated_value > cls._max_limit or calculated_value < cls._min_limit:
+        if (
+            0 < cls._max_limit < calculated_value
+            or calculated_value < cls._min_limit < 0
+        ):
```

Comment:
One question - was the intention here to limit calculated value from dropping below zero?
I'm not fully sure but I think the validator will never throw an error because 0 < cls._max_limit which will always be false and cls._min_limit < 0 which will also always be false?

Reply:
the idea is, as long as the mix and min limits are 0, the validator wont ever throw an error, but, if we change the limits to something different than 0, then the validator will throw an error if the calculated value is outside of the boundaries.